### ISSUE TITLE
`:TexlabForward` `attempt to concatenate a nil value` error

### ISSUE THREAD

**Issue by riazufila** [neovim/nvim-lspconfig#1242]
- `nvim --version`: `v0.6.0-dev+227-gc1f573fbc`
- nvim-lspconfig version(commit hash): `01153c52b3a2c9a27f7c953f3262131aac5a7277`
- What language server (If the problem is related to a specific language server): `texlab`
- Operating system/version: `Arch Linux 5.13.13-zen1-1-zen`

### How to reproduce the problem from neovim startup
```
lspconfig.texlab.setup {
  settings = {
    texlab = {
      build = {
        forwardSearchAfter = true
      },
      chktex = {
        onOpenAndSave = true
      },
      forwardSearch = {
        executable = "zathura",
        args = {"--synctex-forward", "%l:1:%f", "%p"}
      }
    }
  }
}
```

### Actual behaviour
Zathura spawns however upon exiting an error occurs, `Error executing vim.schedule lua callback: ...ack/packer/start/nvim-lspconfig/lua/lspconfig/texlab.lua:40: attempt to concatenate a nil value`

### Expected behaviour
No error spawns after Zathura exits.

### Minimal init.vim or init.lua and code sample
```
-- local modules = {
--   "options",
--   "plugins"
-- }

-- -- Require modules
-- for i = 1, #modules do
--   require(modules[i])
-- end

vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]

local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'

local function load_plugins()
  require('packer').startup {
    {
      'wbthomason/packer.nvim',
      'neovim/nvim-lspconfig',
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. '/plugin/packer_compiled.lua',
    },
  }
end

_G.load_config = function()
  vim.lsp.set_log_level 'trace'
  local nvim_lsp = require 'lspconfig'
  local on_attach = function(_, bufnr)
    local function buf_set_keymap(...)
      vim.api.nvim_buf_set_keymap(bufnr, ...)
    end
    local function buf_set_option(...)
      vim.api.nvim_buf_set_option(bufnr, ...)
    end

    buf_set_option('omnifunc', 'v:lua.vim.lsp.omnifunc')

    -- Mappings.
    local opts = { noremap = true, silent = true }
    buf_set_keymap('n', 'gD', '<Cmd>lua vim.lsp.buf.declaration()<CR>', opts)
    buf_set_keymap('n', 'gd', '<Cmd>lua vim.lsp.buf.definition()<CR>', opts)
    buf_set_keymap('n', 'K', '<Cmd>lua vim.lsp.buf.hover()<CR>', opts)
    buf_set_keymap('n', 'gi', '<cmd>lua vim.lsp.buf.implementation()<CR>', opts)
    buf_set_keymap('n', '<C-k>', '<cmd>lua vim.lsp.buf.signature_help()<CR>', opts)
    buf_set_keymap('n', '<space>wa', '<cmd>lua vim.lsp.buf.add_workspace_folder()<CR>', opts)
    buf_set_keymap('n', '<space>wr', '<cmd>lua vim.lsp.buf.remove_workspace_folder()<CR>', opts)
    buf_set_keymap('n', '<space>wl', '<cmd>lua print(vim.inspect(vim.lsp.buf.list_workspace_folders()))<CR>', opts)
    buf_set_keymap('n', '<space>D', '<cmd>lua vim.lsp.buf.type_definition()<CR>', opts)
    buf_set_keymap('n', '<space>rn', '<cmd>lua vim.lsp.buf.rename()<CR>', opts)
    buf_set_keymap('n', 'gr', '<cmd>lua vim.lsp.buf.references()<CR>', opts)
    buf_set_keymap('n', '<space>e', '<cmd>lua vim.lsp.diagnostic.show_line_diagnostics()<CR>', opts)
    buf_set_keymap('n', '[d', '<cmd>lua vim.lsp.diagnostic.goto_prev()<CR>', opts)
    buf_set_keymap('n', ']d', '<cmd>lua vim.lsp.diagnostic.goto_next()<CR>', opts)
    buf_set_keymap('n', '<space>q', '<cmd>lua vim.lsp.diagnostic.set_loclist()<CR>', opts)
  end

  -- Add the server that troubles you here
  local name = 'texlab'
  local cmd = { 'texlab' } -- needed for elixirls, omnisharp, sumneko_lua
  if not name then
    print 'You have not defined a server name, please edit minimal_init.lua'
  end
  if not nvim_lsp[name].document_config.default_config.cmd and not cmd then
    print [[You have not defined a server default cmd for a server
      that requires it please edit minimal_init.lua]]
  end

  nvim_lsp[name].setup {
    cmd = cmd,
    on_attach = on_attach,
    settings = {
    texlab = {
      build = {
        forwardSearchAfter = true
      },
      chktex = {
        onOpenAndSave = true
      },
      forwardSearch = {
        executable = "zathura",
        args = {"--synctex-forward", "%l:1:%f", "%p"}
      }
    }
  }
}

  print [[You can find your log at $HOME/.cache/nvim/lsp.log. Please paste in a github issue under a details tag as described in the issue template.]]
end

if vim.fn.isdirectory(install_path) == 0 then
  vim.fn.system { 'git', 'clone', 'https://github.com/wbthomason/packer.nvim', install_path }
  load_plugins()
  require('packer').sync()
  vim.cmd [[autocmd User PackerComplete ++once lua load_config()]]
else
  load_plugins()
  require('packer').sync()
  _G.load_config()
end

```

### Health check
<details>
<summary>Checkhealth result</summary>

```
health#lspconfig#check    
========================================================================    
## Checking language server protocol configuration    
  - INFO: html: configuration checked.    
  - INFO: sumneko_lua: configuration checked.    
  - INFO: cssls: configuration checked.    
  - INFO: jsonls: configuration checked.    
  - INFO: texlab: configuration checked.    
  - INFO: bashls: configuration checked.    
  - INFO: yamlls: configuration checked.    
  - INFO: clangd: configuration checked.    
  - INFO: pyright: configuration checked.    
  - INFO: flow: configuration checked.    
  - INFO: tsserver: configuration checked.
```
</details>

### LSP log

<details>
<summary>Log file</summary>

```
paste your log here
```
</details>

**Comment by clason** [comment]
Fixed in #1240

**Comment by riazufila** [comment]
Thanks. I was surprised why the error went away after just a few minutes of making this issue 😅
